_pages/attributes.md

```diff
@@ -177,6 +177,64 @@ Requires the `profile` or `profile:verified_at` scope.
 </td>
 <td markdown="1">
 `verified_at` (string, ISO8601 format)
+</td>
+    </tr>
+    -----------------------------
+    <tr>
+<td markdown="1">
+**x509** <br />
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+`x509` (string) 
+
+Requires the `x509` scope
+</td>
+<td markdown="1">
+`x509`
+</td>
+    </tr>
+    <tr>
+<td markdown="1">
+**x509 Subject** <br />
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+`x509_subject` (string)
+
+Requires the `x509:subject` scope
+</td>
+<td markdown="1">
+`x509_subject`
+</td>
+    </tr>
+    <tr>
+<td markdown="1">
+**x509 Presented** <br />
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+`x509_presented` (string)
```

Comment:
I'm having trouble tracking down an example value for this -- we're sure it's a string right?

Reply:
I'm not sure-- I just added the content @achapm suggested, but if something else seems more correct to you, feel free to put that in.